### DISCUSSION TITLE
Buttons not working

### DISCUSSION THREAD

**Discussion by bvn9ne** [typemill/typemill#440] [Q&A]
I think I need some help. I previously was using TypeMill 1. I backed up and cleared the location on the host. I uploaded and unzipped TypeMill 2.1.1. I had to change the Php version. I have tried it with 8.0, 8.1, and 8.2.

 If I try to save a change or create a page/folder it will not do so. Interestingly, if I click the create page button with it empty, it will give me the error "Special Characters are not allowed. Length between 1 and 40." So, it is reading the button click, it just isn't performing any action for saving.

Hopefully, someone can point me in the right direction. Thank you all.

**Comment by bvn9ne** [comment]
Nevermind, I figured it out. Apparently, it was because of the proxying on cloudflare. I am not sure how this will best be handled for me but that is a different issue and my problem.

**Comment by trendschau** [comment]
Good to hear! If it is a general problem with cloudflare, it would be interesting how you solved the issue. Typemill has a proxy detection (in developer settings), but I don't think that it can help here. 

So information are probably welcome for other users who use cloudflare....
